api/sheets/places_in.md

```diff
@@ -8,37 +8,49 @@ grand_parent: API
 
 # Get Places Contained within Another Place
 
-## `=DCPLACESIN(dcids, placeType)`
+Given a list of parent [`Place`](https://datacommons.org/browser/Place) [DCIDs](glossary.md),
```

Comment:
Does (glossary.md) work or do we need to use /glossary.html?

Reply:
oh yes--I will change that!

Reply:
please check for this post-merge from upstream (there's a conflict)